### ISSUE TITLE
Connection failed - Dio: invalid url

### ISSUE THREAD

**Issue by zagl** [Piwigo/piwigo-flutter-app#5]
In the readme it says that http is not supported yet. That's why I was surprised that it worked nevertheless in version 0.0.3-alpha.

Today I tried 0.0.4-alpha and got the error message Connection failed - Dio: invalid url. I guess it's because I used http. But as it was working in the previous version I am not sure if it might be another issue.

Thank you very much for developing this app. I am very much looking forward to it.

**Comment by remi-martin** [comment]
Thanks for your feedback.

The reason why we mentioned that http was not supported was because it is not supposed to work by default. I have enabled the http right before the first alpha release so I was not sure if it worked well.

Now I think that the way I enabled http requests is the reason of why the app kept crashing for android 8 and bellow.
Let me confirm that ... Then, we will decide between http servers and a working application for Android 6 - 8.
